example/src/test/java/com/example/BUILD.bazel

```diff
@@ -0,0 +1,8 @@
+java_test(
+    name = "Test1",
```

Comment:
this is manually written right?

Reply:
Yea, I haven't hooked up Java support in this PR.